packages/stream_chat_flutter/lib/src/message_input/attachment_picker/stream_attachment_picker.dart

```diff
@@ -824,24 +907,43 @@ Widget webOrDesktopAttachmentPickerBuilder({
         WebOrDesktopAttachmentPickerOption(
           key: 'image-picker',
           type: AttachmentPickerType.images,
-          icon: StreamSvgIcon.pictures(size: 36).toIconThemeSvgIcon(),
+          icon: StreamSvgIcon.pictures().toIconThemeSvgIcon(),
           title: context.translations.uploadAPhotoLabel,
         ),
         WebOrDesktopAttachmentPickerOption(
           key: 'video-picker',
           type: AttachmentPickerType.videos,
-          icon: StreamSvgIcon.record(size: 36).toIconThemeSvgIcon(),
+          icon: StreamSvgIcon.record().toIconThemeSvgIcon(),
           title: context.translations.uploadAVideoLabel,
         ),
         WebOrDesktopAttachmentPickerOption(
           key: 'file-picker',
           type: AttachmentPickerType.files,
-          icon: StreamSvgIcon.files(size: 36).toIconThemeSvgIcon(),
+          icon: StreamSvgIcon.files().toIconThemeSvgIcon(),
           title: context.translations.uploadAFileLabel,
         ),
+        WebOrDesktopAttachmentPickerOption(
+          key: 'poll-creator',
+          type: AttachmentPickerType.poll,
+          icon: StreamSvgIcon.polls().toIconThemeSvgIcon(),
+          title: 'Create a poll',
```

Comment:
Should we add a translation label here too?

Reply:
Nice catch, Added